app/views/api_docs/vendor_api_docs/reference/draft.html.erb

```diff
@@ -40,80 +40,101 @@
 
 <p class="govuk-heading-s">Deferring applications</p>
 <p class="govuk-body">
-  We now support deferring an offer to the next cycle.
+  We now support <%= govuk_link_to 'deferring', '#post-applications-application_id-defer-offer' %> an offer to the next cycle, as well as <%= govuk_link_to 'reinstating', '#post-applications-application_id-reinstate-offer' %> a deferred offer in the next cycle.
+</p>
+<p class="govuk-body">To defer an application, the state of the application will need to either be <code>pending_conditions</code> or <code>recruited</code>, then the <%= govuk_link_to 'defer offer endpoint', '#post-applications-application_id-defer-offer' %> can be invoked. This will change the status of the application to a <b>new</b> status: <code>offer_deferred</code>. It will also populate the fields <code>offer_deferred_at</code> as well as maintain the status before deferral of the application in the field <code>status_before_deferral</code>. Please note that deferred applications did not previously appear in the application lists and will start appearing now when syncing with the API. If an application is deferred it can only be reinstated in the next cycle.
+</p>
+<p class="govuk-body">To reinstate a deferred application in the next cycle, the <%= govuk_link_to 'reinstate offer endpoint', '#post-applications-application_id-reinstate-offer' %> can be invoked. For successfully reinstating an offer, the same course, location and study mode should be present in the new cycle. To determine success the conditions will be required to be set as met or not met, which will transition the state of the application accordingly to <code>recruited</code> or <code>conditions_not_met</code>. Please note that deferred applications from the previous cycle will start appearing in the new cycle when syncing applications, with courses from the previous year initially set until reinstated.
+</p>
+<p class="govuk-body">If an application is reinstated successfully, the new course from the cycle will appear under the <%= govuk_link_to 'OfferObject', '#offer-object' %>.
 </p>
-
 <p class="govuk-heading-s">Withdrawing applications</p>
 <p class="govuk-body">
-  We now support withdrawing an application at the candidate's request.
+  We now support <%= govuk_link_to 'withdrawing', '#post-applications-application_id-withdraw' %> an application at the candidate's request. If an application is withdrawn at a candidate's request, it will either transition to the <code>declined</code> state if the application is in the <code>offer</code> state, or the <code>withdrawn</code> state if it's in any other state. We currently don't require a reason for when an application is withdrawn at a candidate's request.
```

Comment:
I think this path is a bit ambiguous as `withdraw` offer isn't a separate path. consider explicit `withdraw-offer` (ie alias for reject) and `withdraw-application` paths?

Reply:
Also "withdraw an application or decline an offer" is confusing in the docs, I think — doesn't withdrawal cover both, with declining any current offers as a side effect?

Reply:
It is confusing, we have since tried to improve the description. One reason we steered clear of `withdraw-offer` is that we are not tackling normal offer withdrawal. Instead, this is withdrawing an application at a candidate's request, which results either in a withdrawn application or a declined offer.

Reply:
Yes, exactly! this is withdrawing an application, and the other thing called `withdraw` is withdrawing an offer, so intention was to make it explicit what was being withdrawn in each case

Reply:
`withdraw-offer`: the `reject` endpoint currently withdraws an offer OR rejects an application and we are explicitly hiding the `offer_withdrawn` status. We have moved away from separating those into their own endpoints yet because the work for R4R will not be in v1.1. When we do move towards that, we will be adding that endpoint.

`withdraw` vs `withdraw-application`: if we are to move to separating which we will eventually, any action on `/applications` is implied that it is on the application, so our preference is to keep the action itself as is. We are going to test the waters with providers around this endpoint anyway and with adequate explanation of what the endpoint does we're hoping it'll be clear enough.

Reply:
Great. Makes sense